src/extension/content-script/onstart.js

```diff
@@ -0,0 +1,28 @@
+import shouldInject from "./shouldInject";
```

Comment:
Could we try to add new files directly as TS? Even if all types are "FixMe". At least the base is existing.